TombEngine/Objects/TR3/Vehicles/upv.cpp

```diff
@@ -223,17 +223,24 @@ namespace TEN::Entities::Vehicles
 
 	static void FireUPVHarpoon(ItemInfo* UPVItem, ItemInfo* laraItem)
 	{
-		auto* harpoon = FireHarpoon(*laraItem);
-
-		if (harpoon == nullptr)
-			return;
+		// FireHarpoon(*laraItem, nullptr);
```

Comment:
According to coding conventions, you should not leave randomly commented code lines in the codebase. If you are not sure if your new code is stable, more tests should be made.